assets/migrations/generator/generatorConfig.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE generatorConfiguration PUBLIC "-//mybatis.org//DTD MyBatis Generator Configuration 1.0//EN" "http://mybatis.org/dtd/mybatis-generator-config_1_0.dtd">
 <generatorConfiguration>
-	<classPathEntry
-		location="/home/samuel/Projects/opensrp_server/opensrp-server-configs/assets/migrations/drivers/postgresql-42.2.1.jar" />
```

Comment:
this should then read `<project_root>/opensrp-server-configs/assets/migrations/drivers/postgresql-42.2.1.jar`